### ISSUE TITLE
a staff of radiance (magic) (glowing) (Bahamut)

### ISSUE THREAD

**Issue by Awlan** [torilmud/issues#685]
Weapon doesn't give desired effect when it procs:
Powers: Celestial Power
Can affect you as :
       Affects : HITPOINTS by 30
       Affects : HITROLL by 5
Combat Effects : Radiant Pattern

**Comment by shevarash** [comment]
Can you be more specific please?

**Comment by Dartan82** [comment]
I think he means Radiant Pattern doesn't do anything.  Just tried on a bard.

**Comment by Awlan** [comment]
Yeah, it sends a message that it has procced, but the proc doesnt seem to have any effect. damage or otherwise

**Comment by Lilithelle** [comment]
Testing it on test mud, not seeing it proc at all.  I'm only seeing the brilliant enchantment firing.  Its supposed to give -hit -dam to the mob on proc.
